test/e2e/fixture.go

```diff
@@ -254,6 +254,116 @@ func exportLogs(tb testing.TB, data *TestData, logsSubDir string, writeNodeLogs
 	}
 }
 
+func exportMainTestLogs(data *TestData, logsSubDir string, writeNodeLogs bool) {
+	const timeFormat = "Jan02-15-04-05"
```

Comment:
There is a const `timeFormat` in function `exportLogs()`, maybe you could make that one accessible and reuse it.